File: client/employee-management/src/app/login/account.ts
export class Account {
  username: string = '';
  password: string = '';
  isHr: boolean = false;

  constructor(username: string, password: string, isHr: boolean) {
    this.username = username;
    this.password = password;
    this.isHr = isHr;
  }
}

File: AuthServer/src/main/java/com/example/springssoauthserver/domain/Account.java
package com.example.springssoauthserver.domain;

import lombok.*;

/**
 * Helper POJO to parse RequestBody
 * The payload of post request from Angular should be the login credentials
 * {
 *     username: "username",
 *     password: "password",
 *     isHr: true/false
 * }
 */
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Account {
    private String username;
    private String password;
    private String isHr;
}
